src/main/java/com/exactpro/th2/taskutils/BlockingScheduledRetryableTaskQueue.java

```diff
@@ -165,7 +171,7 @@ public ScheduledRetryableTask<V> take() {
         lock.lock();
         try {
             while (true) {
-                if (taskQueue.size() > 0)
+                if (!taskQueue.isEmpty())
                     return taskQueue.poll();
                 else
                     addition.awaitUninterruptibly();
```

Comment:
question: we also have `awaitUninterruptibly` here - should we replace it with `await` as well?